.github/SPRINT_TEMPLATES/sprint-goal.md

```diff
@@ -0,0 +1,70 @@
+name: Sprint goal
+about: Open a new Sprint Goal
+title: ''
+labels: ''
+assignees: ''
+---
+## Description
+
+Write a description of the issue
+
+
+### Goal and Acceptance Criteria
+Release X.Y.Z-rc
+
+[ReleaseVersion] - [DATE] - [Sprint Goal]
+- [DeliveryType][DeliveryDomain] ..delivery description (actor -> action)
+   - Acceptance criteria: // list of acceptance criteria to check on zombienet in Gurken format
```

Comment:
We gotta be careful prescribing zombienet since it is at the top of the testing triangle. We should just write these as tests wherever they need to be. I would say we provide the acceptance criteria for sure in gherkin though.